docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -19,9 +19,9 @@ In this tutorial, you learn how to:
 > [!div class="checklist"]
 > * Understand the problem
 > * Select the appropriate machine learning task
-> * Prepare and understand your data
+> * Prepare and understand the data
```

Comment:
@JRAlexander, what are the conventions for "your" vs "the" for this repo and .NET docs? I see other tutorials are also using phrases like "your data".

Reply:
Our conventions accept both so it's a case by case scenario. I think in this case using the is better.